### PR TITLE
chore: regenerate protos under MomentoProtos module and fix import paths

### PR DESCRIPTION
**Pull request by anitarua**
Wrote a bash script in new folder `generated` to commit regenerated proto code. 
The script also wraps the protos in a module named `MomentoProtos` so that when we rename `SimpleCacheClient` to `CacheClient`, the class won't clash with the previously named module `CacheClient`.

Once the new protos were committed, I also fixed the import paths necessary to get the unit tests passing again (ran using `rspec`)

### CODE REVIEW

**Review by nand4011 — APPROVED** [review]
Looks good to me.

File: lib/momento/generated/generate_protos.sh [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+# Generate the code
+grpc_tools_ruby_protoc -I ./ --ruby_out=./ --grpc_out=./ ./*.proto
+
+# Remove the protos
+rm -f ./*.proto
+
+# In the *_pb.rb files, replace the require with require_relative
+sed -i '' -E "s/require '(.*)_pb'/require_relative '\1_pb'/g" *_pb.rb
+
+# Wrap the generated code in a module named MomentoProtos but only if not "module Scs"
```
Comment (cprice404) [inline lib/momento/generated/generate_protos.sh:30]: what does the `if not "module Scs"` part here mean?

File: lib/momento/generated/generate_protos.sh [hunk 2]
```diff
@@ -0,0 +1,47 @@
...
+for file in *_pb.rb; do
+  sed -i '' -E 's/^module (.*)/module MomentoProtos::\1/g' $file
+done
```
Comment (cprice404) [inline lib/momento/generated/generate_protos.sh:33]: i presume this means that they didn't provide a namespace/module option in the protoc tooling?  :)

File: lib/momento/generated/generate_protos.sh [hunk 3]
```diff
@@ -0,0 +1,47 @@
...
+# In the *_services_pb.rb files, change the module names.
+# * rename ::ControlClient to ::MomentoProtos::ControlClient
+# * rename ::CacheClient to ::MomentoProtos::CacheClient
+for file in *_services_pb.rb; do
+  sed -i '' -E 's/::ControlClient::/::MomentoProtos::ControlClient::/g' $file
+  sed -i '' -E 's/::CacheClient::/::MomentoProtos::CacheClient::/g' $file
+done
+
+# Run the linter
+rubocop -A *_pb.rb
```
Comment (cprice404) [inline lib/momento/generated/generate_protos.sh:47]: this whole thing is gross but seems like a reasonable path forward if they didn't provide what we need in the tooling. thanks for figuring it out.

**Review by cprice404 — APPROVED** [review]
no blockers, just one question about the "scs" comment

File: lib/momento/generated/generate_protos.sh (hunk 1 above)
Comment (anitarua) [reply lib/momento/generated/generate_protos.sh:30]: oops, stale comment from when I was trying to figure out how to wrap only the top-level module in MomentoProtos, will fix

File: lib/momento/generated/generate_protos.sh (hunk 2 above)
Comment (anitarua) [reply lib/momento/generated/generate_protos.sh:33]: not that I could find 😅

File: lib/momento/generated/generate_protos.sh (hunk 3 above)
Comment (anitarua) [reply lib/momento/generated/generate_protos.sh:47]: image

**Review by cprice404 — APPROVED** [review]

**Pull request merged**